persistence/sql/persister_test.go

```diff
@@ -240,8 +239,7 @@ func TestPersister(t *testing.T) {
 			})
 			t.Run("contract=courier.TestPersister", func(t *testing.T) {
 				pop.SetLogger(pl(t))
-				upsert, insert := sqltesthelpers.DefaultNetworkWrapper(p)
```

Comment:
We use this helper elswhere so it's needed to have this little function. Can you please revert those changes? Thanks! :)

Reply:
@aeneasr Alright, i reverted the change. How does it look now ? :)